ember-server/src/test/scala/org/http4s/ember/server/EmberServerSuite.scala

```diff
@@ -56,9 +57,13 @@ class EmberServerSuite extends Http4sSuite {
     FunFixture.map2(server, client)
 
   fixture.test("server responds to requests") { case (server, client) =>
-    IO.sleep(3.seconds) >> client
+    client
       .get(s"http://${server.address.getHostName}:${server.address.getPort}")(_.status.pure[IO])
       .timeout(5.seconds)
       .assertEquals(Status.Ok)
   }
+
+//  server.test("server startup fails if address is already in use") { case _ =>
```

Comment:
I'm leaving this commented because sometimes the new server can't bind to the same port, even though the server has already finalized. Opening a follow up issue